gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -159,6 +159,161 @@
+                      </div>
+
+                      <p class="text-danger" ng-if="webauthn.errors.registrationFailed" translate>Failed to create credential, see Syncthing logs for details.</p>
+
+                      <p class="text-info" ng-if="webauthnAvailable() && isRawIpAddress()" translate>WebAuthn is not supported on raw IP address hosts.</p>
+
+                      <p class="text-info" ng-if="!isLocationInsecure() && !webauthnAvailable()" translate>Your browser does not support WebAuthn.</p>
+
+                      <p class="text-info" ng-if="!locationMatchesWebauthnOrigin()" translate translate-value-address="{{ getWebauthnOrigin() }}">Reload page at {%address%} to enable WebAuthn.</p>
+
+                      <p ng-if="isLocationInsecure() || !locationMatchesWebauthnOrigin()">
+                        <button ng-if="tmpGUI && settingsModified()" type="button" class="btn btn-primary" ng-click="reloadSettingsAtWebauthnAddress(true)"><span class="fas fa-check"></span>&nbsp;<span translate translate-value-address="{{ getWebauthnOrigin() }}">Save settings and reload page at {%address%}</span>
+                        </button>
+
+                        <button ng-if="tmpGUI && !settingsModified()" type="button" class="btn btn-primary" ng-click="reloadSettingsAtWebauthnAddress(false)" translate translate-value-address="{{ getWebauthnOrigin() }}">Reload page at {%address%}</button>
+                      </p>
+                    </td>
+
+                    <td class="small-data button-cell" colspan="1000">
+                      <button type="button" class="btn btn-link btn-sm" ng-click="registerWebauthnCredential()" ng-disabled="!webauthnReady()">
+                        <span class="fas fa-plus"></span>&nbsp;<span translate>Add</span>
+                      </button>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+            </div>
+
+            <div class="table-responsive webauthn-credentials">
+              <table class="table-striped table" ng-repeat="_ in tmpGUI.webauthnState.credentials | filter:{rpId: '!'+tmpGUI.webauthnRpId}:true | limitTo:1">
+                <thead>
+                  <tr>
+                    <th colspan="1000"><label translate>Ineligible WebAuthn Credentials (Passkeys)</label></th>
+                  </tr>
+                  <tr>
+                    <td colspan="1000">
+                      <p translate>The credentials below were created with a different setting of "Webauthn Rp Id" in advanced settings. They cannot be used without restoring this setting to the value shown as "RP ID" below. Note that doing so would make any credentials above ineligible instead.</p>
+                    </td>
+                  </tr>
+                  <tr>
+                    <th><label translate>Name</label></th>
```

Comment:
You missed this one in https://github.com/syncthing/syncthing/commit/4d5b84af835b56f23281a96c3d756ad0d04b846b